README.md

```diff
@@ -109,6 +110,12 @@ For more information please refer to the [search api documentation](https://gith
 
 See the [documentation on the Q&A frontend](docs/q-and-a.md) and how it relates to a finder.
 
+## Changing email signup page defaults
+
+To change the default frequency selected on the email signup, or to subscribe to link-based facets, read the [documentation on email changes](docs/business-finder-email-signup.md).
+
+For more information on what link-based facets are and how they can enable your finder to retrieve content from multiple publishing apps, read [EU Exit Business Readiness finder link-based facets](docs/business-finder-link-based-facets.md).
```

Comment:
I'm not sure this needs to be detailed under `Changing email signup page defaults` as it isn't an email signup page default

Reply:
Where else should it go? I could rename the section to `Customising the email signup page` if that helps...?

Reply:
I'd remove this section as it's a hard-coded thing that only applies to the business finder not something that can be configured at the moment. If we were reading in the default frequency from the YAML, it would be different.

Reply:
Have removed the section and added a line to the list of finders as suggested below.